specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/examples/VpnSiteLinkConnectionGetIkeSas.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2020-08-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "gatewayName": "gateway1",
+    "connectionName": "vpnConnection1",
+    "linkConnectionName": "Connection-Link1"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/westus/operationResults/{operationId}?api-version=2020-08-01"
+      }
+    },
+    "200": {
+      "body": "\"{\"Status\":\"Successful\",\"Data\":null}\""
```

Comment:
Why is the body a string type?

Reply:
Hi @kairu-ms,
The IKE SAs returned are in string format type from NRP. The changes are like that of VNG Connection GetIkeSas